applications/RomApplication/python_scripts/basis_visualization_analysis.py

```diff
@@ -0,0 +1,163 @@
+import KratosMultiphysics
+from KratosMultiphysics.analysis_stage import AnalysisStage
+from KratosMultiphysics.python_solver import PythonSolver
+
+class _RomParametersReaderSolver(PythonSolver):
+    """Pretends to be a solver, but actually just reads the results from the RomParameters.json"""
+
+    def __init__(self, model, settings):
+        super().__init__(model, settings)
+
+        model_part_name = settings["model_part_name"].GetString()
+
+        if self.model.HasModelPart(model_part_name):
+            self.main_model_part = self.model.GetModelPart(model_part_name)
+        else:
+            self.main_model_part = self.model.CreateModelPart(model_part_name)
+
+        settings_filename = self.settings["rom_parameters_file"].GetString()
+        with open(settings_filename, "r") as f:
+            rom_params = KratosMultiphysics.Parameters(f.read())
+
+        self.rom_settings = rom_params
+
+        self.number_of_modes = rom_params["rom_settings"]["number_of_rom_dofs"].GetInt()
+
+        self.variable_names = [p.GetString() for p in rom_params["rom_settings"]["nodal_unknowns"]]
+        self.variables = None
```

Comment:
```suggestion
        self.variable_names = rom_params["rom_settings"].GetStringArray()
        self.variables = kratos_utilities.GenerateVariableListFromInput(rom_params["rom_settings"])
```

https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/kratos_utilities.py